.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+exclude: (.*/vendor/.*)
+
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
```

Comment:
Those are some default hooks that are good to have IMHO. They're not required to run clang-format, though.